NewForce-Cohort-5/tabloidmvc-the-spaghetti-squad
# Ticket #10 - Create a Category

# Description

As an admin, this feature branch will allow the user to be able to create a new category so that it can give authors the ability to better classify their posts. Anytime a user creates a new category, they will be redirected back to the category management page that lists all the categories in alphabetical order. Code additions/changes were made in the following files:

1) CategoryRepository.cs
-a method to add a category to the database was created
2) ICategoryRepository.cs
-with the addition of the add method in the category repository, this same method was also added in this file 
2) CategoryController.cs
-a view was created for the create a category form, where the user can only input a name 

## Type of change

Please delete options that are not relevant.

# Testing Instructions

1) Run the command "git fetch --all" to pull down this new branch locally
2) Run the command "git checkout cr-ticket10" to switch over to this branch for viewing/testing
3) Go ahead and log in as an admin with the email admin@example.com
4) Navigate over to the "Category Management" link in the navbar, click "Create a Category", and proceed to add a new Category by its name
5) After submitting, the user should be redirected back to the list of all categories in the Category Management page
6) To make sure this new category(s) has been added to the list of all categories that is currently being displayed, you can run this SQL query statement to validate results:

SELECT *
FROM Category;

7) Locate the newly added category in the SQL query to validate the display results on the browser window

# Checklist:
